### ISSUE TITLE
Question about "standard depth estimation model" referred in the article

### ISSUE THREAD

**Issue by michaeltan53** [YvanYin/Metric3D#58]
Hi, many thanks for your excellent work. I got two question about the method of obtaining depth.

It is described in Figure 2 saying that "The transformed image Ic is fed into a standard depth estimation model to produce the predicted metric depth Dc in the canonical space. During training, Dc is supervised by a GT depth Dc∗ which is also transformed into the canonical space". Is the standard depth estimation model defined in class BaseDepthModel(nn.Module)? Does this model generate the metric depth of an input image?

And also, how are the ground truth depth Dc∗ images obtained? Thanks a lot.

**Comment by YvanYin** [comment]
We create a space, i.e. canonical space. In this space, all metrics are consistent. In this space, BaseDepthModel will learn the canonical metric, which may have a gap to the real metric. 
GT Dc* is obtained by scaling the original GT Dc.

**Comment by michaeltan53** [comment]
Thanks, one more question, how was the the original ground truth depth obtained, by BaseDepthModel? And what is the specific architecture of BaseDepthModel? I didn't get it in the _base_model_.py file.

Thanks for responding.

**Comment by YvanYin** [comment]
We create a model architecture for the depth model. 
All ground truth depth from the original datasets.
